pandas/core/generic.py

```diff
@@ -3791,7 +3796,9 @@ def values(self):
         int32. By numpy.find_common_type convention, mixing int64 and uint64
         will result in a flot64 dtype.
         """
-        return self.as_matrix()
+        self._consolidate_inplace()
+        bm_axis = self._get_block_manager_axis(axis=1)
+        return self._data.as_matrix(axis=bm_axis)
```

Comment:
instead of this, just pass ``transpose = self._AXIS_REVERSED`` in.

Reply:
But ``_data.as_matrix`` has no `´transpose`` parameter, do you mean ``axis=self._AXIS_REVERSED``?

Reply:
no I mean add the argument ``transpose=False`` rather than ``axis`` prob simpler for now.